### PR TITLE
Fix oreo notificatio banner

### PR DESCRIPTION
**Pull request by aurora-labs**
https://trello.com/c/il197ii9

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -109,8 +109,10 @@ android {
 
 }
 
-ext.androidSupportVersion = '27.0.2'
-ext.butterKnifeVersion = '8.8.1'
+ext {
```
Comment (juanmatm82) [inline app/build.gradle:112]: Nice!!

File: sdk/build.gradle [hunk 2]
```diff
@@ -89,17 +88,27 @@ android {
     useLibrary 'org.apache.http.legacy'
 }
 
+ext {
+    playServicesVersion = '11.8.0'
+    androidSupportVersion = '27.0.2'
+}
+
+configurations.all {
+    resolutionStrategy.force "com.android.support:support-v4:${androidSupportVersion}"
```
Comment (juanmatm82) [inline sdk/build.gradle:97]: Is this the way to solve the compatibility issues which we talk the last day about?

**Review by juanmatm82 — APPROVED** [review]
Ok!

File: sdk/build.gradle (hunk 2 above)
Comment (aurora-labs) [reply sdk/build.gradle:97]: Yes, that's telling Gradle to force a version of the dependency.

**Pull request merged**